modules/appeals_api/app/swagger/appeals_api/v2/api_description.md

```diff
@@ -43,47 +43,48 @@ The **evidence upload** statuses available for simulation are the statuses liste
 
 ### Evidence Uploads
 
-Our NOD evidence submission endpoints allow a client to upload a document package (documents and metadata) of supporting evidence for their submitted NOD by following these steps.
+Our NOD and Supplemental Claim evidence submission endpoints allow a client to upload a document package (documents and metadata) of supporting evidence for their submission by following these steps.
 
-1. Use the POST endpoint [`/notice_of_disagreements/evidence_submissions`] to return a JSON service response with the attributes listed below.
-    - `guid`: An identifier used for subsequent evidence upload status requests (not to be confused with the NOD submission GUID)
-    - `location`: A URL to which the actual document package payload can be submitted in the next step. The URL is specific to this upload request, and should not be re-used for subsequent uploads. The URL is valid for 900 seconds (15 minutes) from the time of this response. If the location is not used within 15 minutes, the GUID will expire. Once expired, status checks on the GUID will return a status of `expired`.
+1. Use the POST endpoint `/notice_of_disagreements/evidence_submissions` or `/supplemental_claims/evidence_submissions` to return a JSON service response with the attributes listed below.
 
-1. Client Request: PUT to the location URL returned in step 1.
-    - Request body should be encoded as binary multipart/form-data, equivalent to that generated by an HTML form submission or using “curl -F…”.
-    - No `apikey` authorization header is required for this request, as authorization is embedded in the signed location URL.
-    - The metadata.json file uploaded to the location URL with the evidence documents MUST set the `businessLine` to “BVA” to ensure your supporting  documents route to the Board of Veterans Appeals (BVA).
-    - The JSON key for the metadata.json file is "metadata", the initial file is "content".  Any subsequent file will be "attachment1", "attachment2", and so forth.
+- `guid`: An identifier used for subsequent evidence upload status requests (not to be confused with the NOD or SC submission GUID)
+- `location`: A URL to which the actual document package payload can be submitted in the next step. The URL is specific to this upload request, and should not be re-used for subsequent uploads. The URL is valid for 900 seconds (15 minutes) from the time of this response. If the location is not used within 15 minutes, the GUID will expire. Once expired, status checks on the GUID will return a status of `expired`.
+2. Client Request: PUT to the location URL returned in step 1.
```

Comment:
Markdown has a nice feature for numbered lists where you can do `1,1,1,1` and it'll automatically increase the numbers for you instead of worrying about keeping `1,2,3,4` in order as you CRUD the list. Let's keep using that here.

Reply:
ok